### PR TITLE
Redux store

### PR DESCRIPTION
**Pull request by danmainah**
**In this milestone I did the following**

- Created redux directories  using Ducks pattern
- Created redux store

### CODE REVIEW

**Review by themonster2015 — APPROVED** [review]
Hi @danmainah  your code works fine on my end. Good job!

File: src/redux/Destinations/destinations.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { getDestinations } from "../../helpers/getDestinationData";
+const FETCH_DATA = 'FETCH_DATA';
+const initialState = [];
+
+export const getCorona = () => async (dispatch) => {
+    const destinations = await getDestinations();
+    dispatch({ type: FETCH_DATA, payload: destinations });
```
Comment (AleksandarIvezic) [inline src/redux/Destinations/destinations.js]: Please fix this misspelled function name to `getDestinations`.

**Review by AleksandarIvezic — APPROVED** [review]
Hi @danmainah 👋🏼

STATUS APPROVED ✔

Your project is complete!
I noticed one misspelled function so please check the comments before merging.

Cheers 🥂 and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by AleksandarIvezic — COMMENTED** [review]
Hey @danmainah, 
Please make sure that you don't overwrite the changes you did in the previous pull request!!!

Cheers!!! 🥂

**Pull request merged**